explorer/explorerroutes.go

```diff
@@ -1165,6 +1164,13 @@ func (exp *explorerUI) AddressPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	pageKey, ok := r.Context().Value(ctxAddressPageKey).(string)
```

Comment:
Please doc that this will/should be one of the defined templates ("address" or "addresstable"), and the purpose of this is to select the template via configurable middleware (see explorerrouter.go).  Something along those lines please.

Reply:
This is outdated now. Now using [the same context with a different explorer function](https://github.com/decred/dcrdata/blob/54ed2a0d3cf9022f08c0fc81e1b9dc99ba7375d7/main.go#L529).